cmd/sqlflowserver/main_test.go

```diff
@@ -1364,6 +1364,33 @@ INTO %s.explain_result;`, caseTestTable, caseInto, caseDB)
 	}
 }
 
+func CaseTrainXGBoostOnPAI(t *testing.T) {
+	a := assert.New(t)
+	trainSQL := fmt.Sprintf(`SELECT * FROM %s
+	TO TRAIN xgboost.gbtree
+	WITH
+		objective="multi:softprob",
+		train.num_boost_round = 30,
+		eta = 0.4,
+		num_class = 3
+	LABEL class
+	INTO my_xgb_classi_model;`, caseTrainTable)
+	_, _, err := connectAndRunSQL(trainSQL)
```

Comment:
This test case is named `TrainXGBoostOnPAI`.  How is the line related to PAI? Is that `connectAndRunSQL` runs SQL statement by default on PAI?  If so, we need a comment at the end of this line like

```go
// connectAndRunSQL runs statements on PAI by default.
```

Otherwise, we might need other explanatory comments.

Reply:
`CaseTrainXGBoostOnPAI` is a sub-test for `TestEnd2EndMaxComputePAI`. Added comments for `TestEnd2EndMaxComputePAI`.